src/pyclass/create_type_object.rs

```diff
@@ -557,8 +556,9 @@ fn bpo_45315_workaround(py: Python<'_>, class_name: CString) {
     std::mem::forget(class_name);
 }
 
-/// Default new implementation
-#[cfg(not(all(Py_3_10, not(PyPy))))]
+/// Default new implementation, to match
+/// <https://github.com/python/cpython/blob/3663b2ad54c9e15775a605facf69da8f5ee8d335/Objects/typeobject.c#L2427-L2428>
```

Comment:
Not sure if this comment still makes sense. I guess the output should be identical still..

--- 

Also I think the `#[cfg(not(PyPy))]` in the trampoline below needs to go now, because PyPy does the same thing  as CPython now, right?

Reply:
Thanks for catching, sorry for my shoddy patch 😂